public/js/index.js

```diff
@@ -1458,29 +1458,26 @@ $(document).ready(function () {
     });
 
     // Helpers.
-    $('.delete-button').click(function () {
-        var $this = $(this);
-        var filter = "";
-        if ($this.attr("id")) {
-          filter += "#"+$this.attr("id")
+    $('.delete-button').click(showDeletePopup);
+
+    var prev_val = "{{branch_name}}";
```

Comment:
It would be better to store dialog template in variable when for the first time used and replace allways `${branchName}` using that template not replace prev_value. For replacement you could reuse my function (not in master, yet):
https://github.com/lafriks/gitea/blob/486a881e8ae2c7fc697379b8f01465e8f0c11509/public/js/index.js#L326